src/core/xml_parser.c

```diff
@@ -1,6 +1,7 @@
 #include "xml_parser.h"
 
 #include "core/log.h"
+#include "game/resource.h"
```

Comment:
Duplicate include, you're already including this in the header

Reply:
I was weak and desperate. I just wanted the thing to compile :'(